src/NITTA/Model/Problems/BindPU.hs

```diff
@@ -0,0 +1,37 @@
+{-# LANGUAGE ConstraintKinds #-}
+{-# LANGUAGE DeriveGeneric #-}
+{-# LANGUAGE FlexibleContexts #-}
+{-# LANGUAGE FlexibleInstances #-}
+{-# LANGUAGE FunctionalDependencies #-}
+{-# LANGUAGE UndecidableInstances #-}
+{-# LANGUAGE NamedFieldPuns #-}
+{-# LANGUAGE OverloadedStrings #-}
+
+{- |
+Module      : NITTA.Model.Problems.Bind
+Description : Function distribution between processor units
+Copyright   : (c) Aleksandr Penskoi, 2019
+License     : BSD3
+Maintainer  : aleksandr.penskoi@gmail.com
+Stability   : experimental
+-}
+module NITTA.Model.Problems.BindPU (
+    BindPU (..),
+    BindPUProblem (..),
+) where
+
+import Data.String.ToString
+import GHC.Generics (Generic)
+
+data BindPU tag = BindPU
+    { bnTag :: tag
+    , puTag :: tag
```

Comment:
Please, add doc. The type looks a little bit strange because (if I correctly understand), we need: network tag (for future), new pu tag, new pu type.